### PR TITLE
MouseDemo: A more visual approach for MouseEvents :^)

### PR DESCRIPTION
**Pull request by phoffmeister**

### CODE REVIEW

**Comment by phoffmeister** [comment]
[](https://postimg.cc/DJ5vMGTP)

**Comment by mrkaragoz** [comment]
That’s a thicc mouse ;)

**Comment by phoffmeister** [comment]
Sorry, it is missing the RGB LEDs :D

**Comment by sunverwerth** [comment]
@mrkaragoz wdym? Looks like a normal mouse to me :^) img

**Comment by bugaevc** [comment]
Could we have a more round mouse, like the one pictured here:



Except viewed from the top?

File: Demos/Mouse/main.cpp [hunk 1]
```diff
@@ -9,97 +9,150 @@
 #include <LibGUI/Widget.h>
 #include <LibGUI/Window.h>
 #include <LibGfx/Bitmap.h>
+#include <LibGfx/Path.h>
 
-static unsigned s_mouse_button_state;
+#include <math.h>
```
Comment (tgsm) [inline Demos/Mouse/main.cpp:14]:
```suggestion
#include <LibM/math.h>
```

Comment (bugaevc) [reply Demos/Mouse/main.cpp:14]: Why? `math.h` is a completely fine (and in fact portable) way of referring to the header

Comment (tgsm) [reply Demos/Mouse/main.cpp:14]: good point!

**Comment by awesomekling** [comment]
This is a nice improvement, especially the mouse wheel thingy :)

@bugaevc feel free to improve roundness :D

**Pull request merged**